torchgen/gen_vmap_plumbing.py

```diff
@@ -132,6 +132,14 @@ def is_mutated_arg(argument: Argument) -> bool:
     return argument.annotation is not None and argument.annotation.is_write
 
 
+def gen_check_escaped(check: str, what: str) -> str:
+    return f"""TORCH_CHECK(
```

Comment:
Can this instead call the `VMAP_CHECK_ESCAPED` macro from above?

Reply:
I checked and it does not include the same header.

Reply:
We control the generated file -- let's add the necessary headers to it so we don't need to repeat ourselves. See `def gen_all_vmap_plumbing` for the headers in the generated file

Reply:
`ATen/functorch/BatchRulesHelper.h` includes `ATen/VmapGeneratedPlumbing.h` should I define the function in `VmapGeneratedPlumbing`?

Reply:
Does in aten/src/ATen/functorch/PlumbingHelper.h work?

Reply:
Yes, that is where I ended up putting it.